scripts/orphaned.py

```
@@ -1,20 +1,59 @@
+    Returns TorrentCategoriesDictionary with all defined categories.
+
+    Args:
+        client: qBittorrent client instance
+        cache_scope: Unique identifier to scope cache per client (use id(client))
+                     to prevent cache contamination across different client instances
+    """
+    _ = cache_scope  # Used by @cached decorator for cache key generation
+    return client.torrent_categories
+
 
 def check_files_on_disk(client, torrents: List, exclude_file_patterns: List[str] = [], exclude_dirs: List[str] = []) -> List[str]:
```

Ruff (0.14.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)